src/Estimators/include/BipedalLocomotion/RobotDynamicsEstimator/SubModel.h

```diff
@@ -73,47 +73,101 @@ class SubModel
     std::vector<std::string> m_externalContactList; /**< List of the additional external contacts */
 
 public:
+    /**
+     * Determines the validity of the object.
+     * @return True if the object is valid, false otherwise.
+     */
+    bool isValid() const;
+
     /**
      * Getters
      */
 
     /**
-     * @brief Access model.
+     * @brief Access m_model.
```

Comment:
I would avoid mentioning `m_model` you can change the documentation in 
```suggestion
     * @brief Get the `iDynTree::Model` instance.
     * @note The actual implementation of the Model is currently stored in an `iDynTree::Model`
```

This is valid also for the others

Reply:
Done